packages/react/src/themes/teams-dark/siteVariables.ts

```diff
@@ -33,7 +33,10 @@ export const green04 = naturalColors.lightGreen[900]
 //
 // SHADOW LEVELS
 //
-export const shadowLevel1 = '0 .2rem .4rem -.075rem rgba(255, 255, 255, 0.1)'
+export const shadowLevel1 = '0 .2rem .4rem -.075rem rgba(0, 0, 0, .1)'
```

Comment:
where are these values coming from? can you reference a design file / spec in a private chat?

Reply:
Pinged in private.